patches/0013-p256-armv8-optimizations

```diff
@@ -593,6 +595,19 @@ Index: aws-lc/third_party/boringssl/crypto/fipsmodule/ec/p256.c
 +  }
 +}
 +
++static int ecp_nistz256_scalar_to_montgomery_inv_vartime(const EC_GROUP *group,
++                                                 EC_SCALAR *out,
++                                                 const EC_SCALAR *in) {
++  assert(group->order.width == P256_LIMBS);
++  if (!beeu_mod_inverse_vartime(out->words, in->words, group->order.d)) {
```

Comment:
Can we be explicit here? I.e.  `FAIL_VALUE == beeu_mod_inverse_vartime(out->words, in->words, group->order.d)`

That is more readable.

Reply:
Same comment as above (l.601)